TechliftTelegramBot/Services/AgencyInfo.cs

```diff
@@ -0,0 +1,39 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Telegram.Bot.Types;
+using TechliftTelegramBot.Models;
+
+
+namespace TechliftTelegramBot.Services
+{
+    
+    public class AgencyInfo
```

Comment:
class name and file name should end with service or whatever it represents so in this case a better name is AgencyInfoService

Reply:
yes changed